docs/plugins/subsonicupdate.rst

```diff
@@ -12,6 +12,12 @@ Then, you'll probably want to configure the specifics of your Subsonic server.
 You can do that using a ``subsonic:`` section in your ``config.yaml``,
 which looks like this::
 
+    subsonic:
+        url: https://mydomain.com:443/subsonic
```

Comment:
Let's use the officially sanctioned domain for examples:
```suggestion
        url: https://example.com:443/subsonic
```

Reply:
Didn't even know that was a thing. Thanks!